src/main/java/VASSAL/build/module/map/KeyBufferer.java

```diff
@@ -123,13 +123,13 @@ public void mousePressed(MouseEvent e) {
     if (p != null && !ignoreEvent) {
       boolean movingStacksPickupUnits = (Boolean) GameModule.getGameModule().getPrefs().getValue(Map.MOVING_STACKS_PICKUP_UNITS);
       if (!kbuf.contains(p)) {
-        if (!e.isShiftDown() && !e.isControlDown()) {
+        if (!e.isShiftDown() && !SwingUtils.isControlDown(e)) {
```

Comment:
I think to get KeyBufferer to work you're actually going to have to either
(a) add an "&& SwingUtils.notOnAMacintosh()" filter to this, OR
(b) Go ahead and start using Command on a Mac and CTRL on everything else, for the purpose of KeyBufferer only.